beserial/src/lib.rs

```diff
@@ -198,7 +198,7 @@ impl Deserialize for bool {
 
 impl Serialize for bool {
     fn serialize<W: WriteBytesExt>(&self, writer: &mut W) -> Result<usize, SerializingError> {
-        writer.write_u8(if *self { 1u8 } else { 0u8 })?;
+        writer.write_u8(u8::from(*self))?;
```

Comment:
This doesn't make the code clearer to me. Maybe ignore the warning for this line if you agree.

Reply:
I agree. Added an ignore to the warining